src/main/kotlin/dev/kord/x/emoji/DiscordEmoji.kt

```diff
@@ -33,31 +33,40 @@ sealed class DiscordEmoji {
         override val unicode: String
             get() = "$code${tone.unicode}"
 
+        /**
+         * Checks [other] to be the same emote but ignores [tone].
+         */
+        fun isSimilar(other: Any?): Boolean {
```

Comment:
Why use `Any?` here? The function can only return true if `other` is a `DiscordEmoji`.